### PR TITLE
Feature/code cleaning

### PR DESCRIPTION
**Pull request by davidBentoPereira**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
-    <!-- Favicon -->
-    <link rel="icon" href="<%= image_path 'calculate_vert_icon160.png' %>" />
     <!-- Custom CSS -->
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
     <%= csrf_meta_tags %>
+    <!-- Favicon -->
+    <%= favicon_link_tag 'calculate_vert_icon160.png' %>
```
Comment (Nicolas-Hermet) [inline app/views/layouts/application.html.erb:12]: Bueno ça !

File: app/views/customers/index.html.erb [hunk 2]
```diff
@@ -1,70 +1,68 @@
...
+        <ol class="breadcrumb mb-0">
+          <li class="breadcrumb-item"><a href="<%= root_path %>">Accueil</a></li>
+          <li class="breadcrumb-item active" aria-current="page">Liste de mes clients</li>
+        </ol>
+      </nav>
+      <!-- Page Title -->
+      <h1 class="mt-3">Liste de mes clients</h1>
     </div>
+    <!-- Button add Customer-->
+    <a href="<%= new_customer_path %>" class="btn btn-primary ml-3">Nouveau client</a>
```
Comment (Nicolas-Hermet) [inline app/views/customers/index.html.erb]: hum hum

File: app/views/customers/index.html.erb [hunk 3]
```diff
@@ -1,70 +1,68 @@
...
-            </tr>
-          <% end %>
-          </tbody>
-        </table>
-      </div>
+            <td><%= customer.company_name %></td>
+            <td><%= customer.email %></td>
 
+            <td class="text-center">
+              <a href="<%= customer_path(customer.id) %>" class="btn btn-primary"><i class="fa fa-eye"></i></a>
```
Comment (Nicolas-Hermet) [inline app/views/customers/index.html.erb]: hum hum

File: app/views/customers/index.html.erb [hunk 4]
```diff
@@ -1,70 +1,68 @@
...
+            </td>
+            <td class="text-center">
+              <a href="<%= edit_customer_path(customer.id) %>" class="btn btn-info"><i class="fa fa-pen"> Editer</i></a>
```
Comment (Nicolas-Hermet) [inline app/views/customers/index.html.erb]: hum hum

File: app/views/customers/show.html.erb [hunk 5]
```diff
@@ -1,50 +1,48 @@
...
+          <li class="breadcrumb-item"><a href="<%= root_path %>">Accueil</a></li>
+          <li class="breadcrumb-item active" aria-current="page"><a href="<%= customers_path %>">Liste de mes clients</a></li>
+        </ol>
+      </nav>
+      <!-- Page Title -->
+      <h1 class="mt-3">Mon client</h1>
+    </div>
+    <!-- Button "Back to customer index" -->
+    <div class="nav justify-content-end">
+      <a href="<%= customers_path %>" class="btn btn-secondary"><i class="fa fa-arrow-left"></i> Retour accueil</a>
```
Comment (Nicolas-Hermet) [inline app/views/customers/show.html.erb]: hum hum

File: app/views/customers/show.html.erb [hunk 6]
```diff
@@ -1,50 +1,48 @@
...
-        <div class="col">
-          <%= form_tag customer_path(@customer.id), method: "delete" do %>
-            <button class="btn btn-danger">Supprimer</button>
-          <% end %>
-        </div>
+  <!-- Buttons action -->
+  <div class="row justify-content-center my-5">
+    <div class="row justify-content-center">
+      <div class="col">
+        <a class='btn btn-info btn-block' href="<%= edit_customer_path(@customer) %>">Editer</a>
```
Comment (Nicolas-Hermet) [inline app/views/customers/show.html.erb]: hum hum

File: app/views/devise/passwords/new.html.erb [hunk 7]
```diff
@@ -1,34 +1,34 @@
...
-<div class="layout-login-centered-boxed__form card">
-  <div class="d-flex flex-column justify-content-center align-items-center mt-2 mb-5 navbar-light">
-    <a href="<%= flowdash_path(:index) %>" class="navbar-brand flex-column mb-2 align-items-center mr-0" style="min-width: 0">
-      
-      <span>Mot de passe oublié ?</span>
-    </a>
-  </div>
+  <div class="layout-login-centered-boxed__form card">
+    <div class="d-flex flex-column justify-content-center align-items-center mt-2 mb-5 navbar-light">
+      <a href="<%= flowdash_path(:index) %>" class="navbar-brand flex-column mb-2 align-items-center mr-0" style="min-width: 0">
```
Comment (Nicolas-Hermet) [inline app/views/devise/passwords/new.html.erb]: hum hum

File: app/views/devise/sessions/new.html.erb [hunk 8]
```diff
@@ -1,59 +1,59 @@
...
-    </a>
-  </div>
-
-  <% if alert.present? %>
-    <div class="alert alert-soft-danger d-flex" role="alert">
-      <i class="material-icons mr-3 align-self-center">error</i>
-      <span><%= alert %></span>
+  <div class="layout-login-centered-boxed__form card">
+    <div class="d-flex flex-column justify-content-center align-items-center mt-2 mb-5 navbar-light">
+      <a href="<%= root_path %>" class="navbar-brand flex-column mb-2 align-items-center mr-0" style="min-width: 0">
```
Comment (Nicolas-Hermet) [inline app/views/devise/sessions/new.html.erb]: hum hum

File: app/views/layouts/_footer.html.erb [hunk 9]
```diff
@@ -1,114 +1,63 @@
...
-
     </div>
-    <!-- Grid row -->
-
   </div>
-  <!-- Footer Links -->
 
   <!-- Copyright -->
   <div class=" footer-copyright text-center text-white-50 py-3">© 2020 Copyright:
     <a class="greenblue" href="https://monka-project.herokuapp.com/"> Monka.com</a>
```
Comment (Nicolas-Hermet) [inline app/views/layouts/_footer.html.erb]: hum hum

File: app/views/layouts/_navbar.html.erb [hunk 10]
```diff
@@ -1,36 +1,36 @@
-<!--monka home button -->
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <!-- Monka home button -->
   <a class="navbar-brand" href="<%= root_path %>">monka</a>
```
Comment (Nicolas-Hermet) [inline app/views/layouts/_navbar.html.erb]: hum hum

File: app/views/layouts/_navbar.html.erb [hunk 11]
```diff
@@ -1,36 +1,36 @@
...
 
-
-  <!--navigation -->
+  <!-- Navigation -->
   <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
     <div class="navbar-nav">
       <% if user_signed_in? %>
-        <!--change color for user -->
+        <!-- Change color for user -->
         <a class="nav-item nav-link <%= "active" if controller_name === 'users' && action_name === 'show' %>" href="<%= user_path(current_user) %>">
```
Comment (Nicolas-Hermet) [inline app/views/layouts/_navbar.html.erb]: hum hum

File: app/views/layouts/_navbar.html.erb [hunk 12]
```diff
@@ -1,36 +1,36 @@
...
           Mon profil
           <% if controller_name === 'users' && action_name === 'show' %>
             <span class="sr-only">(current)</span>
           <% end %>
         </a>
-        <!--change color for view index customer -->
+        <!-- Change color for view index customer -->
         <a class="nav-item nav-link <%= "active" if controller_name === 'customers' && action_name === 'index' %>" href="<%= customers_path %>">
```
Comment (Nicolas-Hermet) [inline app/views/layouts/_navbar.html.erb]: hum hum

File: app/views/layouts/_navbar.html.erb [hunk 13]
```diff
@@ -1,36 +1,36 @@
...
           Mes clients
           <% if controller_name === 'customers' && action_name === 'index' %>
             <span class="sr-only">(current)</span>
           <% end %>
         </a>
-        <!--change color for view index quotes -->
+        <!-- Change color for view index quotes -->
         <a class="nav-item nav-link <%= "active" if controller_name === 'quotes' && action_name === 'index' %>" href="<%= quotes_path %>">
```
Comment (Nicolas-Hermet) [inline app/views/layouts/_navbar.html.erb]: hum hum

File: app/views/devise/passwords/new.html.erb [hunk 14]
```diff
@@ -3,8 +3,7 @@
 
   <div class="layout-login-centered-boxed__form card">
     <div class="d-flex flex-column justify-content-center align-items-center mt-2 mb-5 navbar-light">
-      <a href="<%= flowdash_path(:index) %>" class="navbar-brand flex-column mb-2 align-items-center mr-0" style="min-width: 0">
-
+      <a href="<%= root_path %>" class="navbar-brand flex-column mb-2 align-items-center mr-0" style="min-width: 0">
```
Comment (Nicolas-Hermet) [inline app/views/devise/passwords/new.html.erb]: hum hum

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (davidBentoPereira) [reply app/views/layouts/application.html.erb:12]: :+1:

**Pull request merged**